### PR TITLE
Fixed ISO keyboard layout for the Mysterium rev1.1 board

### PR DESCRIPTION
**Pull request by HalfHeuslerAlloy**
## Description

Swapped the key mapping so the the left \ key is bound correctly and that some rows are no longer shifted. Also fixed a mistake when copying this fixed from local drive in earlier pull request.

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* #11705

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/coseyfannitutti/mysterium/mysterium.h [hunk 1]
```diff
@@ -84,6 +84,6 @@
     { K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, _x_, K1F, K1G, K1H }, \
```
Comment (fauxpark) [inline keyboards/coseyfannitutti/mysterium/mysterium.h:84]: K01 above this line should be `XXX`, no?

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/coseyfannitutti/mysterium/mysterium.h (hunk 1 above)
Comment (HalfHeuslerAlloy) [reply keyboards/coseyfannitutti/mysterium/mysterium.h:84]: I'm new to QMK, I'm just following the rest of the header file format. Is xxx more common?

Comment (fauxpark) [reply keyboards/coseyfannitutti/mysterium/mysterium.h:84]: Here it is defined as `_x_`, but the point is the same.

Comment (HalfHeuslerAlloy) [reply keyboards/coseyfannitutti/mysterium/mysterium.h:84]: Yes your right, I forgot to edit that for the 7u

**Review by fauxpark — APPROVED** [review]

**Comment by fauxpark** [comment]
@coseyfannitutti ?

**Comment by HalfHeuslerAlloy** [comment]
I wanted to have these GRAVE_EASCAPE changes in a different pull request, I'll work that out if I can. But anyway GRAVE_ESCAPE is probably not useful when all mysterium keyboards have esc keys. Having grave escape prevents doing ctrl+shift+esc to bring up task manager on windows.

**Review by drashna — APPROVED** [review]

**Pull request merged**